src/components/ResultsEmpty.tsx

```diff
@@ -11,6 +12,8 @@ type ResultsEmptyProp = {
   title?: string
   body?: string
   ctaHref?: string | null
+  ctaText?: string
+  buttonProps?: ButtonProps
```

Comment:
Same here, only passed in `<PriceTable />` currently and feels like this would be static styles if you want to keep it a constant in this component and remove the prop

Reply:
Because this has become more than just Defi made sense to allow us to customize the button thats rendered. We could also just allow us pass in a custom component instead to override the existing button...